server/sonar-web/src/main/js/apps/marketplace/App.tsx

```diff
@@ -161,6 +165,25 @@ export default class App extends React.PureComponent<Props, State> {
       () => {}
     );
 
+  fetchEditions = () => {
+    this.setState({ loadingEditions: true });
+    getEditionsList(this.props.editionsUrl).then(
+      editionsPerVersion => {
+        if (this.mounted) {
+          this.setState({
+            editions: getEditionsForVersion(editionsPerVersion, this.props.sonarqubeVersion),
+            loadingEditions: false
+          });
+        }
+      },
+      () => {
+        if (this.mounted) {
+          this.setState({ loadingEditions: false });
+        }
+      }
+    );
+  };
+
   updateEditionStatus = (editionStatus: EditionStatus) =>
     this.setState({ editionStatus: editionStatus });
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Expected property shorthand in object literal ('{editionStatus}'). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3498)